apps/openmw/mwinput/inputmanagerimp.cpp

```diff
@@ -1034,8 +1036,40 @@ namespace MWInput
 
     void InputManager::screenshot()
     {
-        mScreenCaptureHandler->setFramesToCapture(1);
-        mScreenCaptureHandler->captureNextFrame(*mViewer);
+        bool regularScreenshot = true;
+        bool screenshotTaken = false;
+
+        std::string settingStr;
+
+        try
+        {
+            settingStr = Settings::Manager::getString("screenshot type","Video");
+            regularScreenshot = settingStr.size() == 0 || settingStr.compare("regular") == 0;
+        }
+        catch (std::runtime_error)
```

Comment:
This looks suspicious. We just don't ignore exceptions (unless they are related to an error that has already been handled elsewhere). What are we trying to catch here?